File: naTanjir/naTanjir.Model/StavkeNarudzbe.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace naTanjir.Model
{
    public class StavkeNarudzbe
    {
        public int StavkeNarudzbeId { get; set; }

        public int Kolicina { get; set; }

        public decimal Cijena { get; set; }

        public int NarudzbaId { get; set; }

        public int ProizvodId { get; set; }

        public int RestoranId { get; set; }
        public Proizvod Proizvod { get; set; }
        public Restoran Restoran { get; set; }

    }
}

File: naTanjir/UI/natanjir_desktop/lib/models/stavke_narudzbe.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:natanjir_desktop/models/korisnici.dart';
import 'package:natanjir_desktop/models/narudzba.dart';
import 'package:natanjir_desktop/models/proizvod.dart';
import 'package:natanjir_desktop/models/restoran.dart';

part 'stavke_narudzbe.g.dart';

@JsonSerializable()
class StavkeNarudzbe {
  int? stavkeNarudzbeId;
  int? kolicina;
  double? cijena;
  int? narudzbaId;
  int? proizvodId;
  int? korisnikId;
  int? restoranId;
  Proizvod? proizvod;
  Restoran? restoran;
  StavkeNarudzbe();

  factory StavkeNarudzbe.fromJson(Map<String, dynamic> json) =>
      _$StavkeNarudzbeFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$StavkeNarudzbeToJson(this);
}
